20xt/src/car/src/adc_driver.cpp

```diff
@@ -0,0 +1,102 @@
+#include <iostream>
+#include <mqueue.h>
+#include <vector>
+#include <unistd.h>
+
+#include "ipc_config.h"
+#include "baja_live_comm.pb.h"
+#include "adc.hpp"
+
+#include <chrono>
+
+
+using std::chrono::duration_cast;
+
+#define NUMADC 4
+
+std::string serializeDoubleToBinaryString(double value) {
+    // Cast the address of 'value' to a pointer to 'unsigned char'
+    const unsigned char* p = reinterpret_cast<const unsigned char*>(&value);
+    // Construct a string from the binary data of 'value'
+    return std::string(p, p + sizeof(double));
+}
+
+// We want to produce a series of dummy Communcation values to the TX queue
+int main () {
```

Comment:
@aarenclwong @TravdaTrav As mentioned, this exec seems to be maxing out its core. Maybe because it's spinning as fast as possible? We may need to decrease spin rate